### DISCUSSION TITLE
Add genre or style group

### DISCUSSION THREAD

**Discussion by laurentopia** [SupaGruen/SD-CheatSheet#11] [Ideas]
For example: cute, robots, landscapes, trippy.
We'll fill it.

**Comment by MalikKillian** [comment]
I want this reference to be expanded but I think those keywords are much more dependent on the model. Trippy, for one, is kind of a feeling you get from looking at a picture but I don't think there's any kind of "trippy" style that art historians would agree on. I prefer this cheatsheet to be a bit more objective so it actually has general educational value from an art perspective.

**Reply by laurentopia** [reply]
that's why it has to be crowdsourced.
reference: game genre labeling on steam
